bin/role2docker

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
+
+set -euo pipefail
+set -x
+
+if [ "$1" = '-d' ]; then
+    DEBUG=1
+    shift
+else
+    DEBUG=0
+fi
+
+ROLE=$1
+TOP=$(cd $(dirname $0)/..; pwd)
+
+if [ -z "$ROLE" ]; then
+    echo "Usage: $0 <role>" 1>&2
+    exit 1
+fi
+
+cd $TOP
+
+if [ ! -d roles/$ROLE ]; then
+    echo "No role $ROLE" 1>&2
+    exit 1
+fi
+
+if [ ! -d roles/$ROLE/docker ]; then
+    echo "No role docker support in $ROLE" 1>&2
+    exit 1
+fi
+
+PLAYBOOK=$(mktemp -p $TOP/roles/$ROLE/docker playbookXXXXXX.yml)
+BASENAME=$(basename $PLAYBOOK .yml)
+BASE=
+
+cleanup() {
+    set +e
+    rm -f $PLAYBOOK
+
+    if [ -n "$BASE" -a "$DEBUG" -eq 0 ]; then
+        docker stop $BASE
+        docker rm $BASE
+        docker rmi -f $BASE
+    fi
+}
+
+trap cleanup 0
+
+# Build base image if it doesn't exist
+if ! docker image inspect --format='{{ .Id }}' wazo-platform/base; then
+    cat > $PLAYBOOK <<EOF
+---
+- hosts: localhost
+  become: yes
+  roles:
+    - role: preflight_checks
+    - role: debian_upgrade_first
+    - role: debian-repo-wazo
```

Comment:
nitpicking: is it possible to be uniform between - and _ ?

Reply:
agreed but we should do that in another PR as these names don't come from this PR...